packages/assistant/src/RPA/Assistant/flet_client.py

```diff
@@ -218,28 +146,29 @@ def display_flet_window(
         timeout: int,
     ):
         self._show_flet(
-            self._create_flet_target_function(),
-            title,
-            height,
-            width,
-            on_top,
-            location,
+            self._create_flet_target_function(title, height, width, on_top, location),
             timeout,
         )
 
     def clear_elements(self):
         if self.page:
-            self.page.controls.clear()
+            if self.page.controls:
+                self.page.controls.clear()
             self.page.overlay.clear()
             self.page.update()
         self._elements.visible.clear()
         self._elements.invisible.clear()
 
-    def update_elements(self, page: Page):
+    def update_elements(self):
         """Updates the UI and shows new elements which have been added into the element
         lists
         """
-        return self._create_flet_target_function(page)(None)
+        assert self.page
```

Comment:
don't really like this, because this will throw a rather random assertion exception, maybe catch it and raise something different instead?

Reply:
Yeah good point.